### PR TITLE
Add docs for check_version

### LINKED ISSUE #15: Add an example into the docs
Some examples in the actual code would be very useful

See the example of examples :) in https://docs.rs/hyper/0.13.5/hyper/struct.Body.html#example

### PR DESCRIPTION
**Pull request by tarikeshaq**
fixes #15

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -154,11 +146,27 @@ fn compare_with_latest(
...
 }
 
-/// Validates current version of the crate
-/// Takes the current name and version
-/// Takes a std::time::Duration to determine the interval
-///         So that it does not notify the user every run
-/// Prints directly to stdout (Will probably change to be more asynchrounos)
+/// Checks if there exists an update by checking against crates.io and notifies the user by printing to stdout
+///
+/// Takes:
```
Comment (vladikoff) [inline src/lib.rs]: this doesn't format that well it seems via `cargo doc`

image

Comment (tarikeshaq) [reply src/lib.rs]: Yikes, actually probably just gonna keep that one line and remove the arguments. they should be self explanatory

Comment (vladikoff) [reply src/lib.rs]: @tarikeshaq try this:

```diff
diff --git a/src/lib.rs b/src/lib.rs
index d036650..8a20790 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -148,13 +148,11 @@ fn compare_with_latest(
 
 /// Checks if there exists an update by checking against crates.io and notifies the user by printing to stdout
 ///
-/// Takes:
+/// # Arguments:
 ///
-///   name: &str representing the name of the crate, you can use env!("CARGO_PKG_NAME")
-///
-///   current_version: &str the version of the CLI, use env!("CARGO_PKG_VERSION")
-///
-///   interval: std::time::Duration representing the interval duration
+///   * `name` - representing the name of the crate, you can use `env!("CARGO_PKG_NAME")`
+///   * `current_version` - the version of the CLI, use `env!("CARGO_PKG_VERSION")`
+///   * `interval` - std::time::Duration representing the interval duration
 ///
 /// # Examples
 ///

```

Comment (vladikoff) [reply src/lib.rs]: Example here: https://doc.rust-lang.org/stable/rust-by-example/meta/doc.html

Comment (tarikeshaq) [reply src/lib.rs]: That makes sense 🤦

**Review by vladikoff — APPROVED** [review]
docs look perfect !

**Pull request merged**